src/core/electric_utility.jl

```diff
@@ -195,6 +200,51 @@ struct ElectricUtility
         cambium_emissions_region = "NA - Cambium data not used for climate emissions" # will be overwritten if Cambium is used
         
         if !is_MPC
+            # Initialize clean energy fraction series
```

Comment:
I think you'll want to add the off_grid_flag check here as well (same as done for the grid emissions)

Reply:
Off-grid flag to be added to "scenario.jl"